packages/compass-crud/src/components/bulk-update-dialog.tsx

```diff
@@ -0,0 +1,306 @@
+          severity: 'error',
+          from: syntaxError.loc.index,
+          to: syntaxError.loc.index,
+        },
+      ];
+    }
+
+    return [];
+  }, [syntaxError]);
+
+  return (
+    <FormModal
+      title={title}
+      subtitle={ns}
+      size="large"
+      open={isOpen}
+      onSubmit={runBulkUpdate}
+      onCancel={closeBulkUpdateDialog}
+      cancelButtonText="Close"
+      submitButtonText="Update documents"
+      submitDisabled={!!(syntaxError || serverError)}
+    >
+      <div className={columnsStyles}>
+        <div className={queryStyles}>
+          <div className={queryFieldStyles}>
+            <TextInput
+              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+              // @ts-ignore the label can be any component, but it's weirdly typed to string
+              label={
+                <QueryLabel
+                  label="Filter"
+                  tooltip="Return to the Documents tab to edit this query."
+                />
+              }
+              disabled={true}
+              value={(toJSString(filter) ?? '').replace(/\s+/g, ' ')}
+            />
+          </div>
+
+          <div className={cx(queryFieldStyles, updateFieldStyles)}>
+            <Label htmlFor="bulk-update-update">Update</Label>
+            <Description className={descriptionStyles}>
+              <Link href="https://www.mongodb.com/docs/manual/reference/method/db.collection.updateMany/#std-label-update-many-update">
+                Learn more about Update syntax
+              </Link>
+            </Description>
+            <KeylineCard
+              className={cx(
+                codeContainerStyles,
+                multilineContainerStyles,
+                darkMode ? codeDarkContainerStyles : codeLightContainerStyles
+              )}
+            >
+              <CodemirrorMultilineEditor
+                text={text}
+                onChangeText={onChangeText}
+                id="bulk-update-update"
+                data-testid="bulk-update-update"
+                onBlur={() => ({})}
+                className={codeEditorStyles}
+                annotations={annotations}
+              />
+
+              <div className={bannerContainerStyles}>
+                {syntaxError && (
+                  <Banner
+                    variant={BannerVariant.Warning}
+                    className={bannerStyles}
+                  >
+                    {syntaxError.message}
+                  </Banner>
+                )}
+                {serverError && !syntaxError && (
+                  <Banner
+                    variant={BannerVariant.Danger}
+                    className={bannerStyles}
+                  >
+                    {serverError.message}
+                  </Banner>
+                )}
+              </div>
+            </KeylineCard>
+          </div>
+        </div>
+        <div className={previewStyles}>
+          <Label htmlFor="bulk-update-preview">
```

Comment:
In the designs this preview label has enough padding so that the filter preview on the left and the first doc on the right are aligned:

![image](https://github.com/mongodb-js/compass/assets/5036933/06788379-e3db-4477-83df-4372c13fa236)

Doesn't seem to be the case here:

![image](https://github.com/mongodb-js/compass/assets/5036933/b56664c2-444a-46d2-9758-1a825594c54a)

Reply:
OK it is that info icon. I made it just "hang out" now rather than push that line bigger because that seems like the simplest fix.